src/app/calendar/calendar.component.scss

```diff
@@ -0,0 +1,86 @@
+@import url('https://fonts.googleapis.com/css2?family=ABeeZee&family=Comfortaa:wght@300;400;500;600;700&display=swap');
+
+
+@mixin table-entries(){
```

Comment:
please move mixin's to a global mixin file and import it, unless it's a one time use only, then we don't need a mixin for just one usage